File: Manual 06/Question02.cpp
#include <iostream>
using namespace std;

// Function to draw a rectangle of stars with given dimensions
void drawRectangle(int rows, int cols)
{
    for (int i = 1; i <= rows; i++)
    {
        for (int j = 1; j <= cols; j++)
        {
            cout << "* ";
        }
        cout << endl;
    }
}

int main()
{
    int rows, cols;
    cout << "Enter Number of Rows: ";
    cin >> rows;
    cout << "Enter Number of Columns: ";
    cin >> cols;

    // Check for invalid input
    if (rows < 0 || cols < 0)
    {
        cout << "Rectangle cannot be drawn with negative dimensions!" << endl;
    }
    else
    {
        drawRectangle(rows, cols);
    }

    return 0;
}

File: Python/Manual 03/Question02.py
def print_hourglass(n):
    i = 0
    while i < n:
        j = n - 1
        while j >= n - i:
            print(" ", end="")
            j -= 1

        k = 1
        while k <= n - i:
            print("* ", end="")
            k += 1
        print()
        i += 1

    i = 2
    while i <= n:
        k = 1
        while k <= n - i:
            print(" ", end="")
            k += 1

        j = 1
        while j <= i:
            print("* ", end="")
            j += 1
        print()
        i += 1


n = int(input("Enter Number of Rows: "))
print_hourglass(n)
